tests/application/cms/test_page_service.py

```diff
@@ -412,6 +412,30 @@ def test_create_new_version_of_page_duplicates_dimension_categorisations(self):
         assert new_link.includes_all is True
         assert new_link.includes_unknown is True
 
+    def test_create_new_minor_version_duplicates_uploads(self):
+        user = UserFactory(user_type=TypeOfUser.RDU_USER)
+        measure_version = MeasureVersionWithDimensionFactory(latest=True)
```

Comment:
If we don't need a dimension here, we can probably just use `MeasureVersionFactory` to be a bit more efficient.

Reply:
Ah of course, uploads just need a measure.  Will fix it.